examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -149,40 +161,54 @@
 # the scores and then threshold it:
 
 from nilearn.image import threshold_img
+
 cut_score = np.mean(scores, axis=0)
 cut_score[cut_score < 0] = 0
 
 # bring the scores into the shape of the background brain
 score_map_img = masker.inverse_transform(cut_score)
 
-thresholded_score_map_img = threshold_img(score_map_img, threshold=1e-6, copy=False)
+thresholded_score_map_img = threshold_img(
+    score_map_img, threshold=1e-6, copy=False
+)
+
+from nilearn.image import coord_transform
```

Comment:
Same here

Reply:
I have tried to move the import in sensible place and re ran isort to make sure it was happy with it.